lib/ui/Dataview.mjs

```diff
@@ -331,10 +337,13 @@ function dataviewToolbar(_this) {
     .map((key) => mapp.ui.utils[_this.dataview]?.toolbar[key]?.(_this))
     .filter((item) => !!item);
 
+  // Create an element for the toolbar buttons
+  _this.btn_row = mapp.utils.html.node`<div class="btn-row">${toolbarElements}</div>`
```

Comment:
Can we use camelCame 🐫 as I think it's the style we want to start perpetuating

Reply:
Resolved in [0f21251](https://github.com/GEOLYTIX/xyz/pull/1740/commits/0f2125196d797256a7966b36d67ffe9d518cc85b)